fastlane/lib/fastlane/actions/modify_services.rb

```diff
@@ -10,7 +10,7 @@ def self.run(params)
           require 'produce/service'
           services = params[:services]
 
-          enabled_services = services.select { |_k, v| v == true || v.to_s == 'on' }.map { |k, v| [k, 'on'] }.to_h
+          enabled_services = services.select { |_k, v| v == true || v.to_s != 'off' }.map { |k, v| [k, v == true || v.to_s == 'on' ? 'on' : v] }.to_h
```

Comment:
My Ruby is to bad to understand this. Can someone explain this line in plain English what is going on here?

Reply:
select { |_k, v| v == true || v.to_s != 'off' } -> similar to filter, it select all entries (key, value pairs) where value is true OR value converted to string is different from "off". This condition matches values: true, "on", :on, "complete", "other_string".

map { |k, v| [k, v == true || v.to_s == 'on' ? 'on' : v] } -> converts values of key-value pair to either "on" or "string". Values true, "on" and :on it converts value to "on". Values "complete", "other_string" are kept the same.